resources/assets/uad_lists.json

```diff
@@ -15923,7 +15923,7 @@
   {
     "id": "com.boundax.koreapreloadappinstaller",
     "list": "Oem",
-    "description": "서비스 앱 설치 (Install the Services app), Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
+    "description": "Install the Services app, Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```

Comment:
I still think "Install the Service app" is not a correct name of an app, but more like a literal translation. Any alternatives for that?

Reply:
Cant think of a better name for it the apk filename is skt-appinstaller.apk i can share the apk file if you want